cmd/gtfToBed/testdata/testOut.bed

```diff
@@ -0,0 +1,7 @@
+chr1	173477335	173488815	transcript	0	+	refGene	.	+	.	gene_id "PRDX6"; transcript_id "NM_004905"; gene_name "PRDX6";
```

Comment:
This might confuse some people because there is a set way to encode genes in bed format and this output we are testing against is not it.  Maybe this is a code review question?  I think we are starting to call things beds that are not beds.  Maybe we should come up with our own name and then keep beds strictly beds?
https://genome.ucsc.edu/FAQ/FAQformat.html#format1